src/get_muskrat_data.R

```diff
@@ -0,0 +1,56 @@
+# Vangstoverzicht muskusrat op basis van GBIF
+
+# packages
+library(sf)
+library(rgbif)
+library(dplyr)
+library(tidyr)
+library(readr)
+
+# load data
+down_musk <- occ_download(
+  pred("taxonKey", 5219858),
+  pred_in("datasetKey", c("ddd51fa5-97ce-48ff-9a58-a09d7e76b103",
```

Comment:
any suggestions how ?

Reply:
There is a field in the GBIF export called "Sampling Protocol"
For example in the "VMM - Rat control occurrences in Flanders, Belgium" this is coded as either "RAT TRAP" or "casual observation". Although I think a lot of catches are also marked as casual observation, so again this would need to be checked within the data or with Dimi or Lien

Reply:
So drop casual observation when dataset == "VMM - Rat control occurrences in Flanders, Belgium" ?

Reply:
If it is correct in the dataset, but I'm really not sure about this.